cmd/demo-client/main.go

```diff
@@ -159,3 +203,49 @@ func Watch(c *cli.Context, client client.Client) error {
 	}
 	return nil
 }
+
+func newPrometheusBridge(address string, gateway string, pushIntervalSec int64) prometheus.Registerer {
+	b := &prometheusBridge{
+		address:         address,
+		pushIntervalSec: pushIntervalSec,
+	}
+	if gateway != "" {
+		b.pusher = push.New(gateway, "drand_client_observations_push")
+		go b.pushLoop()
+	}
+	if address != "" {
+		http.Handle("/metrics", promhttp.Handler())
```

Comment:
should be an explicit collector. avoid the global one.
(The library will call `Register` and give you a collector. Currently this bridge stands in front of the implict global registry. You can make a `NewRegistery()`, that you then can ask for a [`promhttp.HandlerFor(registery)`](https://godoc.org/github.com/prometheus/client_golang/prometheus/promhttp#HandlerFor)
if you use the provided implementations you shouldn't have to re-implement shims for Register/mustregister/unregister